core/src/test/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingServiceSpec.scala

```diff
@@ -21,42 +23,93 @@ import org.scalatestplus.mockito.MockitoSugar
 import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, Future}
 import scala.jdk.CollectionConverters._
+import scala.math.BigDecimal.RoundingMode
 
 
 class SpendReportingServiceSpec extends AnyFlatSpecLike with TestDriverComponent with MockitoSugar with Matchers with MockitoTestUtils {
-  val fields: List[Field] = List(
-    Field.of("cost", StandardSQLTypeName.STRING),
-    Field.of("credits", StandardSQLTypeName.STRING),
-    Field.of("currency", StandardSQLTypeName.STRING),
-    Field.of("date", StandardSQLTypeName.STRING)
-  )
-  val firstRow: List[FieldValue] = List(
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "0.0"), // cost
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "0.0"), // credits
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "USD"), // currency
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, s"${DateTime.now().toString}") // date
-  )
-  val secondRow: List[FieldValue] = List(
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "0.10111"), // cost
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "0.0"), // credits
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, "USD"), // currency
-    FieldValue.of(FieldValue.Attribute.PRIMITIVE, s"${DateTime.now().toString}") // date
-  )
-  val defaultFieldValues: List[FieldValueList] = List(
-    FieldValueList.of(firstRow.asJava, fields:_*),
-    FieldValueList.of(secondRow.asJava, fields:_*)
-  )
-  val defaultPage: PageImpl[FieldValueList] = new PageImpl[FieldValueList](null, null, defaultFieldValues.asJava)
-  val schema: Schema = Schema.of(fields:_*)
-  val defaultTableResult: TableResult = new TableResult(schema, 2, defaultPage)
+  object SpendReportingTestData {
+    object Daily {
```

Comment:
Test data was starting to get unwieldy, so I've moved it into these objects. The test data in `Daily` is mostly the same as what existed before, but I've extracted a few values into variables (e.g. `firstRowCost` and `firstRowDate`) and added the `...CostRounded` variables to use in test assertions. 

The test data in `Workspace` has fields specific to workspace breakdowns, but is otherwise very similar to the `Daily` test data.